service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/FailedPBAPaymentEventHandler.java

```diff
@@ -27,32 +36,72 @@
     private final FailedPBAPaymentContentProvider notificationContent;
 
     @EventListener
-    public void notifyLocalAuthority(FailedPBAPaymentEvent event) {
+    public void notifyApplicant(FailedPBAPaymentEvent event) {
+        CaseData caseData = event.getCaseData();
+        String applicant = defaultIfNull(event.getApplicantName(), "");
```

Comment:
i would move this applicant var into else section, as it does not matter for C110A_APPLICATION